secp256k1-sys/depend/secp256k1/ci/ci.sh

```diff
@@ -80,7 +80,7 @@ esac
     --enable-examples="$EXAMPLES" \
     --enable-ctime-tests="$CTIMETESTS" \
     --with-valgrind="$WITH_VALGRIND" \
-    --host="$HOST" $EXTRAFLAGS
+    --host="$HOST" "$EXTRAFLAGS"
```

Comment:
In bb3f5003535b9cecee763f36ee9eaef9528260ac:

If you want to change this file you'll need to PR upstream, or change the vendoring script here to modify it using `sed`. But you can't edit vendored code inline like this.

Reply:
FTR I believe `$EXTRAFLAGS` is unquoted on purpose.